util/make_timeline.py

```diff
@@ -427,6 +431,13 @@ def main(args):
         default=[],
         help="Combatant names to ignore, e.g. Aratama Soul",
     )
+    parser.add_argument(
+        "-oc",
+        "--only-combatant",
+        nargs="*",
+        default=[],
+        help="Only the listed combatants will generate timeline data, e.g. Aratama Soul",
```

Comment:
Sorry, I think quotes are better here.  (And maybe they're needed elsewhere in the file).  You can't just pass `-oc Aratama Soul` as-is.